drivers/hmis_external_apis/app/views/layouts/hmis_external_apis/external_forms.haml

```diff
@@ -12,6 +12,8 @@
       %script{src: "https://www.google.com/recaptcha/api.js?render=#{page_config.recaptcha_key}"}
 
     %link(rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css" integrity="sha384-xOolHFLEh07PJGoPkLv1IbcEPTNtaed2xpHsD9ESMhqIYd0nLMwNLD69Npy4HI+N" crossorigin="anonymous")
+    %link(rel="stylesheet" href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css" integrity="sha256-p4NxAoJBhIIN+hmNHrzRCf9tD/miZyoHS5obTRR9BMY=" crossorigin="")
+    %script(src="https://unpkg.com/leaflet@1.9.4/dist/leaflet.js" integrity="sha256-20nQCchB9co0qIjJZRGuk2/Z9VM+kNiyxNV1lvTlZBo=" crossorigin="")
```

Comment:
It would be preferable to _only_ include the leaflet js if this is for a form that collects geolocation (aka, only if the geolocation partial is included). Looks doable, I think: https://stackoverflow.com/a/10226010

Reply:
Thanks yes good point 🙏